### PR TITLE
Removing the heading on events page

### PR DESCRIPTION
**Pull request by amanpatel123**
### What are you trying to accomplish?
- Just a minor modification on the index page of events
- Removing the Headings 

### How are you accomplishing it?
 - Not applicable

### Is their anything reviewers should know?
 - This is not related to this MR but I thought I would share anyways. So I came across a gem called `rubycritic`. As the name suggests it critics our codebase. Here's what I got:

image

There's still room for improvement. How do you feel about including this gem in our Gem file? We can also use `guard` to run it every time we modify our code. 

- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -51,7 +50,8 @@
   <% end %>
 <% else %>
   <br>
-  <h4>Upcoming events</h4>
+ 
+  <%= "Unfortunately we don't have any upcomming CSS EVENT currently, we will keep you posted when there's any event" if @future_events.empty? %>
```
Comment (amanpatel123) [inline app/views/events/index.html.erb]: If you can think of some better line then please feel free to suggest. I was thinking we can also have a text box there if there's no future event that would ask anyone visiting the event page to suggest to us CSS of an event they would like CSS to have in the coming months. How do you feel about it @EricPickup

Comment (HarshdipD) [reply app/views/events/index.html.erb]: Good idea

Comment (EricPickup) [reply app/views/events/index.html.erb]:
```suggestion
  <%= "There are no future events scheduled at the moment. Please check back later." if @future_events.empty? %>
```

@amanpatel123 this message is good enough for now.  We'll hopefully never get to the point where there are no scheduled events so I wouldn't bother going through the work of making a suggestion box

**Comment by EricPickup** [comment]
Feel free to add that gem in another PR @amanpatel123 👍

**Comment by amanpatel123** [comment]
Awesome! Thanks for such positive responses 🙏

**Pull request merged**